src/Schema/Table.php

```diff
@@ -256,36 +256,38 @@ public function addUniqueIndex(array $columnNames, $indexName = null, array $opt
      */
     public function renameIndex($oldName, $newName = null)
     {
-        $oldName           = $this->normalizeIdentifier($oldName);
+        $normalizedOldName = $this->normalizeIdentifier($oldName);
         $normalizedNewName = $this->normalizeIdentifier($newName);
 
-        if ($oldName === $normalizedNewName) {
+        if ($normalizedOldName === $normalizedNewName) {
             return $this;
         }
 
-        if (! $this->hasIndex($oldName)) {
-            throw SchemaException::indexDoesNotExist($oldName, $this->_name);
+        if (! $this->hasIndex($normalizedOldName)) {
+            throw SchemaException::indexDoesNotExist($normalizedOldName, $this->_name);
         }
 
         if ($this->hasIndex($normalizedNewName)) {
             throw SchemaException::indexAlreadyExists($normalizedNewName, $this->_name);
         }
 
-        $oldIndex = $this->_indexes[$oldName];
+        $oldIndex = $this->_indexes[$normalizedOldName];
```

Comment:
This and the `hasIndex()` above duplicate the implementation of `getIndex()`. Would it make sense to just call it?